examples/amp-ad-exit.amp.html

```diff
@@ -88,5 +97,16 @@ <h1 on="tap:exit-api.exit(target='target1')" role="button" tabindex="1">amp-ad-e
   </div>
   <p><a href="https://www.google.com/" on="tap:exit-api.exit(target='target1', _elem='anchor')">&lt;A&gt; tags don't work well (try middle-clicking)</a>
 </div>
+
+<div class="ad" on="tap:exit-api.exit(target='target1')" role="button" tabindex="1">
```

Comment:
Nit: Tabindex 1 already in use on line 91

Reply:
tabindex doesn't need to be unique - it's more like a class than id
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex